modules/search/class.jetpack-search.php

```diff
@@ -887,21 +924,16 @@ function convert_wp_es_to_es_args( array $args ) {
 			unset( $es_query_args['sort'] );
 		}
 
-		if ( ! empty( $filters ) && is_array( $filters ) ) {
-			foreach ( $filters as $filter ) {
-				$builder->add_filter( $filter );
-			}
-
-			$es_query_args['filter'] = $builder->build_filter();
-		}
+		Jetpack_Search::score_query_by_recency( $parser );
```

Comment:
Wondering if we should remove this function and place the code in here. It doesn't do much. Should probably also have some filters added, but probably better in a separate PR.